mobile/components/artist_app_banner/index.coffee

```diff
@@ -0,0 +1,25 @@
+AppBanner = require '../app_banner/app_banner.coffee'
+{ USER_AGENT, modal } = require('sharify').data
+
+class ArtistAppBanner extends AppBanner
+    template: ->
+      $ require('./index.jade') { modal: modal }
+    
+    @shouldDisplay: ->
```

Comment:
This is an interesting feature for attaching the method to the class—allowing for `ArtistAppBanner.shouldDisplay` below.

From [the docs](http://coffeescript.org/#classes)

> Finally, class definitions are blocks of executable code, which make for interesting metaprogramming possibilities. Because in the context of a class definition, this is the class object itself (the constructor function), you can assign static properties by using @property: value, and call functions defined in parent classes: @attr 'title', type: 'text'